### ISSUE TITLE
Compiling the solution with .net

### ISSUE THREAD

**Issue by CBenghi** [reefpicker/AquaTracker#1]
Hello,
I've been asked to review the work for Joss (https://github.com/openjournals/joss-reviews/issues/635#issuecomment-383530452), but before I accept I want to be sure that I have the toolset required.

I have not used VB6 in more than a decade, and I don't expect to be able to get it now.
I have access Microsoft Imagine subscription but I cant' see VB6 in the marketplace anymore.
Is there a way to upgrade the project to .net?

Can you advise?
Claudio

**Comment by reefpicker** [comment]
No. It will take a really long time. I have thought of doing it but it
takes more than I have.

Years ago vb6 learning edition was free for 60days.

Do you have to test compile for review?

**Comment by CBenghi** [comment]
You can have a look of an example of the review checklist at:
https://github.com/openjournals/joss-reviews/issues ... any of the review issues has the checks.
In my view, a few of them would expect the reviewer to have access to the development environment.

Can you suggest a download link for vb6 learning edition?

**Comment by reefpicker** [comment]
Unfortunately, VB 6.0 is no longer available from anyone... You would have
to buy it on EBAY...

I guess I will have to find some other way to get this published...

:(

Thanks!

**Comment by CBenghi** [comment]
I'm trying to see if conversion is possible, but some files seem missing:

Project AquaTracker presents the following missing files(s):
```
Class=cBmp; ..\..\..\..\..\avi creator\cBmp.cls
Class=cAVICreator; ..\..\..\..\..\avi creator\cAVICreator.cls
Class=cVideoHandlers; ..\..\..\..\..\avi creator\cVideoHandlers.cls
Class=cVideoHandler; ..\..\..\..\..\avi creator\cVideoHandler.cls
```

My advice would be to review the folder structure of the project with the .vbp in a top directory and the classes and dependencies below.

**Comment by reefpicker** [comment]
Interesting... Let me check...

**Comment by CBenghi** [comment]
If you add the four files to the repo I'll try to fix the vbp.
Claudio

**Comment by reefpicker** [comment]
Sorry I took this long but I finally added the files to the repository!
Thanks!!!

**Comment by CBenghi** [comment]
Hello @reefpicker,
I'm sorry but I've hit another problem with tabclt32.ocx missing.
I think it's installed by vb6, but it does not get distributed by default in windows 7 and 10.
That would have to be part of the distribution.

**Comment by CBenghi** [comment]
Hi @reefpicker,
can you add the missing ocx to the repository?
Thanks,
Claudio
